updatePassword.php

```diff
@@ -0,0 +1,17 @@
+<?php
+session_start();
+$password = $_POST["password"];
+$password = hash('sha256',$password);
+if (isset($_SESSION["forgotPasswordEmail"])) {
+  $email = $_SESSION["forgotPasswordEmail"];
+}
```

Comment:
If the `SESSION` variable `forgotPasswordEmail` is not set, then redirect back to `index.php` with a message `Session timed out. Please try again`.